sql/localsql/schema/migrations/0010_atxs.sql

```diff
@@ -0,0 +1,12 @@
+--- Table for storing blobs of published ATX for regossiping purposes.
+CREATE TABLE atx_blobs
+(
+    id         CHAR(32),
+    pubkey     CHAR(32) NOT NULL,
+    epoch      INT NOT NULL,
+    atx        BLOB,
+    version    INTEGER
+);
+
+CREATE UNIQUE INDEX atx_blobs_id ON atx_blobs (id);
```

Comment:
When creating a new table from scratch, you can just specify `id` as the primary key column